packs/hello-universe-1.0.0/manifests/hello-universe.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: hello-universe-service
+spec:
+  type: LoadBalancer
+  selector:
+    app: hello-universe
+  ports:
+  - protocol: TCP
+    port: 8080
+    targetPort: 8080
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hello-universe-deployment
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: hello-universe
+  template:
+    metadata:
+      labels:
+        app: hello-universe
+    spec:
```

Comment:
We should specify CPU and memory resources in the `spec.containers` block. Try the below values because I'm pretty sure our container requires very few resources. You can see the resource utilization in a deployed cluster 😄
That way, we can also document it in the README. 

```
   resources:
      requests:
        cpu: "100m" # requesting 100 milliCPU
      limits:
        cpu: "200m" # limiting to 200 milliCPU
  ```